logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -342,6 +344,23 @@ sealed class Event(open val id: String, open val transient: Boolean) {
         ) : Conversation(id, transient, conversationId) {
             override fun toLogMap(): Map<String, Any?> = mapOf(typeKey to "Conversation.CodeDeleted")
         }
+
+        data class TypingIndicator(
+            override val id: String,
+            override val conversationId: ConversationId,
+            override val transient: Boolean,
+            val senderUserId: UserId,
+            val timestampIso: String,
```

Comment:
Suggestion: maybe `instant` instead of `timestampIso`? Easier to manipulate and we only need to convert it once.

Reply:
This I'll keep as is, I think the patterns of the project are more important and with this, all the rest of the events are using the same var name/type. Currently, we are just using it for datadog/logging, so no manipulation.